adjust.go

```diff
@@ -80,6 +80,32 @@ func AdjustSaturation(img image.Image, percentage float64) *image.NRGBA {
 	})
 }
 
+// AdjustHues changes the hue of the image using the shift parameter (measured in degrees) and returns the adjusted image.
+// The shift must be in the range (-180, 180).
+// The shift = 0 gives the original image.
+// The shift = 180 (or -180) corresponds to a 180° degree rotation of the color wheel and thus gives the image with its hue inverted for each pixel.
+//
+// Examples:
+//  dstImage = imaging.AdjustHues(srcImage, 90) // Shift Hue by 90°.
+//  dstImage = imaging.AdjustHues(srcImage, -30) // Shift Hue by -30°.
+//
+func AdjustHues(img image.Image, shift float64) *image.NRGBA {
```

Comment:
I've noticed the function name change (`AdjustHue` -> `AdjustHues`). I'm not a native English speaker, so I've tried searching GitHub:
- "adjustHue" -> 48K code results
- "adjust_hue" -> 117K code results
- "adjustHues" -> 10 code results
- "adjust_hues" -> 2 code results

It seems that the former form is much more common.

Reply:
That's a good point! I am also not a native speaker, so I looked at the wikipedia article for "Hue" and they always talk about an image's hue**s**, for example in [the caption of this image](https://en.wikipedia.org/wiki/Hue#/media/File:Hue.gif). So you are right, if everyone else calls it "AdjustHue" than that's probably the way to go.

Reply:
Fixed by commit 84be08e